ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano.hs

```diff
@@ -1,8 +1,13 @@
-{-# LANGUAGE DataKinds           #-}
-{-# LANGUAGE GADTs               #-}
-{-# LANGUAGE KindSignatures      #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TypeOperators       #-}
+{-# LANGUAGE DataKinds             #-}
```

Comment:
I think we can drop some of these pragmas -- especially the `-Wno-orphans` now that `Protocol` is gone.